src/resources/TunnelOfLove.ts

```diff
@@ -41,3 +38,76 @@ export const LovEnamorang = new Copier(
   () => couldUseLoveEnamorang(),
   () => getLovEnamorangMonster()
 );
+
+type LOVEquipment = "LOV Eardigan" | "LOV Epaulettes" | "LOV Earring";
+type LOVEffect = "Lovebotamy" | "Open Heart Surgery" | "Wandering Eye Surgery";
+type LOVExtra =
+  | "LOV Enamorang"
+  | "LOV Emotionizer"
+  | "LOV Extraterrestrial Chocolate"
+  | "LOV Echinacea Bouquet"
+  | "LOV Elephant"
+  | "toast"
+  | null;
```

Comment:
Why null here but not the other options? In case you're going for the trophy or something

Reply:
These are all the options, I think (null means take nothing). Trophy support will have to come later.